packages/react/src/side-nav/SideNavGroup.tsx

```diff
@@ -2,14 +2,14 @@ import { ReactNode } from 'react';
 import { Box } from '../box';
 import { LinkListContext, useLinkListDepth } from './context';
 
-type SideNavGroupProps = { children: ReactNode };
+type SideNavGroupProps = { children: ReactNode; isOpen: boolean };
 
-export function SideNavGroup({ children }: SideNavGroupProps) {
+export function SideNavGroup({ children, isOpen }: SideNavGroupProps) {
 	const depth = useLinkListDepth();
 	const value = depth + 1;
 	return (
 		<LinkListContext.Provider value={value}>
-			<Box as="ul">{children}</Box>
+			{isOpen && <Box as="ul">{children}</Box>}
```

Comment:
If it's not open, should we not render the Provider either?

Reply:
- [ ] conditionally render provider as well